spec/index.bs

```diff
@@ -5317,6 +5345,14 @@ dictionary GPUExternalTextureDescriptor
 
                     1. Let |result| be a new {{GPUExternalTexture}} object wrapping |data|.
 
+                1. If |source| is a {{GPUTextureView}}, then when
+                    |source|.{{GPUTextureView/[[texture]]}}.{{GPUTexture/[[destroyed]]}} is `true`,
```

Comment:
IMHO we should be more explicit and have a WeakSet in the internal slots of GPUTexture. Otherwise the spec isn't as much an "algorithm" than a prose describing what happens.

Reply:
Should we have a WeakSet in the internal slots for `HTMLVideoElement` and `VideoFrame` as well?